contracts/RiskManagerV1.sol

```diff
@@ -298,6 +314,110 @@ contract RiskManagerV1 is IRiskManager, Auctioneer, Ownable {
         auctionLengthChangeInitiated = 0;
     }
 
+    /// @notice Begins the liquidation notifier reward amount update process.
+    /// @dev Can be called only by the contract owner.
+    /// @param _newLiquidationNotifierRewardAmount New liquidation notifier
+    ///        reward amount.
+    function beginLiquidationNotifierRewardAmountUpdate(
```

Comment:
I'd skip the *Amount* part from function names and parameters. Meaning is the same and they can be shorter.